### PR TITLE
DAO-384 Added RIF-Token contract

### PR DESCRIPTION
**Pull request by Freshenext**
Notes:
Had to add RIFTokenTest.sol to solhint ignore list because it was complaining: 
**Compiler version ^0.4.24 does not satisfy the ^0.8.0 semver requirement**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,3 +15,7 @@ node_modules
 
 # Hardhat Ignition default folder for deployments against a local node
 ignition/deployments/chain-31337
+
+# PHPStorm
+
+.idea
```
Comment (rodrigoncalves) [inline .gitignore]: Add a newline at end of file here.

File: .solhintignore [hunk 2]
```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+contracts/RIFTokenTest.sol
```
Comment (rodrigoncalves) [inline .solhintignore]: And here.

File: contracts/RIFTokenTest.sol [hunk 3]
```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.4.24;
+
+import "rif-token-contracts/contracts/RIF/RIFToken.sol";
+// File must exist, else test/RIFTokenTest.ts will fail
+contract RIFTokenTest is RIFToken {
+    constructor () RIFToken () {
+        
+    }
+}
```
Comment (rodrigoncalves) [inline contracts/RIFTokenTest.sol]: And here.

**Review by rodrigoncalves — APPROVED** [review]
Looks perfect. Just add some newline at end of some files and we're good to go.

**Review by rodrigoncalves — CHANGES_REQUESTED** [review]
Actually it's better to rename this file to `RIFToken.test.ts`.

**Comment by Freshenext** [comment]
> Actually it's better to rename this file to `RIFToken.test.ts`.

Yep! Done

**Review by rodrigoncalves — APPROVED** [review]
LG

**Review by sleyter93 — APPROVED** [review]
Great job.

**Review by jessgusclark — APPROVED** [review]

**Pull request merged**